From 299b149b7b47779f6f183605eb3ae14ba67cd709 Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Wed, 5 Oct 2022 15:43:01 +0900
Subject: [PATCH] Call `getrandom` from sys/random.h

TODO(b/190018559): third_party/lss/linux_syscall_support.h is not
provided at current libchrome tree.

Change-Id: Ifc96beb4987ff027ab32f885687e88dbd7c8dc34
patch-name: long-term-0900-Call-getrandom-from-sys-random.h.patch
---
 base/rand_util_posix.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/base/rand_util_posix.cc b/base/rand_util_posix.cc
index 37dd2d32ea..50f7aea8db 100644
--- a/base/rand_util_posix.cc
+++ b/base/rand_util_posix.cc
@@ -28,9 +28,9 @@
 #include "base/time/time.h"
 #include "build/build_config.h"
 
-#if (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)) && !BUILDFLAG(IS_NACL)
+#if (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)) && !BUILDFLAG(IS_NACL) && 0
 #include "third_party/lss/linux_syscall_support.h"
-#elif BUILDFLAG(IS_MAC)
+#elif BUILDFLAG(IS_MAC) || 1
 // TODO(crbug.com/40641285): Waiting for this header to appear in the iOS SDK.
 // (See below.)
 #include <sys/random.h>
-- 
2.45.0.rc0.197.gbae5840b3b-goog

